.vitepress/config.mts

```diff
@@ -0,0 +1,48 @@
+import { defineConfig } from 'vitepress'
+
+export default defineConfig({
+  title: "CS2 KZ Docs",
+  description: "CS 2 Kreedz documentation",
+  head: [['link', { rel: 'icon', href: '/public/favicon.ico' }]],
+  srcDir: "docs",
+
```

Comment:
Enable [VitePress clean urls](https://vitepress.dev/reference/site-config#cleanurls) as GitHub Pages natively supports it.
Will allow for urls to be valid without the `.html` suffix.

```suggestion

cleanUrls: true
```